src/clj/game/cards/resources.clj

```diff
@@ -1973,6 +1974,18 @@
     :leave-play (req (remove-watch state :theophilius-bagbiter)
                      (swap! state assoc-in [:runner :hand-size :base] 5))}
 
+   "Thunder Art Gallery"
+   (let [ability {:once :per-turn
+                  :optional {:prompt (msg "Use Thunder Art Gallery to install a card?")
```

Comment:
I prefer cards like this (optional into card selection) to be a single selection prompt, with the `:cancel-effect` being used as the "No" ability. Players know that clicking Done without having selected a card is the same as "no/cancel". The prompt can then be worded "Install a card with Thunder Art Gallery?".